cmd/create/oidcconfig/cmd.go

```diff
@@ -445,6 +452,14 @@ func (s *CreateUnmanagedOidcConfigAutoStrategy) execute(r *rosa.Runtime) {
 		r.Reporter.Warnf("Secret ARN: %s\tBucketUrl: %s", secretARN, bucketUrl)
 		os.Exit(1)
 	}
+	if output.HasFlag() {
+		err = output.Print(oidcConfig)
```

Comment:
Curious about the spinner, should it be stopped before printing, or will printing interrupt it?

Reply:
Spinner is not created if we have the output flag. However, I think we do need to revisit a lot of code that uses spinner to have some unified check error and stop spinner setup

Reply:
We can track it under this - https://issues.redhat.com/browse/SDA-8847